### ISSUE TITLE
Unable to quickly skim or write test YAML files due to repetitive XPath boilerplate in defect count/severity/line checks

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#1340]
I cannot skim or write test YAML files in `src/test/resources/org/eolang/lints/packs/` quickly because most of them repeat the same raw XPath boilerplate to check defect count, severity, and line number, e.g.:

```yaml
asserts:
  - /defects[count(defect[@severity='error'])=1]
  - /defects/defect[@line='3']
```

We already simplified the "no defects" case with a plain `defects: 0` key. I want the same treatment for the severity/line checks, so `defects:` becomes the single place to describe every expected defect, no XPath required. Target shape:

Single defect:
```yaml
defects:
  - severity: warning
    line: 1
```

Multiple defects, different lines/severities:
```yaml
defects:
  - severity: error
    line: 3
  - severity: error
    line: 7
```

Zero defects (stays exactly as today):
```yaml
defects: 0
```

Count only, no line/severity constraints (rare case, stays exactly as today):
```yaml
defects: 3
```

There are currently 315 files using a raw `severity` XPath assertion and 135 using a raw `line` XPath assertion, so this needs to cover all of them.

Relevant classes to investigate: `XtDefects`, `LtByXslTest`, `PkByXslTest`.

**Comment by volodya-lombrozo** [comment]
This one was solved in #1341

**Comment by 0crat** [comment]
@volodya-lombrozo Thanks for closing this issue! You've earned +8 points for this: +8 as a basis; +0 for 1 days of delay. Your running score is [+336](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
